frontend/src/forms/manage-restaurant-form/MenuItemInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button } from '@/components/ui/button';
+import {
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { useFormContext } from 'react-hook-form';
+
+type Props = {
+  index: number;
+  removeMenuItems: () => void;
+};
+
+const MenuItemInput = ({ index, removeMenuItems }: Props) => {
+  const { control } = useFormContext();
+
+  return (
+    <div className="flex flex-row items-end gap-2">
+      <FormField
+        control={control}
+        name={`menuItems.${index}.name`}
+        render={({ field }) => (
+          <FormItem>
+            <FormLabel className="flex items-center gap-1">
+              Name <FormMessage />
+            </FormLabel>
+            <FormControl>
+              <Input
+                {...field}
+                placeholder="Cheese Pizza"
+                className="bg-white"
+              />
+            </FormControl>
+          </FormItem>
+        )}
+      />
+
+      <FormField
+        control={control}
+        name={`menuItems.${index}.price`}
+        render={({ field }) => (
+          <FormItem>
+            <FormLabel className="flex items-center gap-1">
+              Price <FormMessage />
+            </FormLabel>
+            <FormControl>
+              <Input {...field} placeholder="8.0" className="bg-white" />
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/30#issuecomment-2386850165>#4f441a</a><span> - 10/01/2024, 07:41 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The price input field lacks proper type validation and formatting for numeric input.
</div>
<div id="fix">
<b>Fix:</b> Modify the price input field to use a numeric input type and add pattern validation for decimal numbers.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -47,7 +47,13 @@
            Price <FormMessage />
          </FormLabel>
          <FormControl>
 -          <Input {...field} placeholder="8.0" className="bg-white" />
 +          <Input
 +            {...field}
 +            type="number"
 +            step="0.01"
 +            min="0"
 +            placeholder="8.0"
 +            className="bg-white"
 +          />
          </FormControl>
        </FormItem>
      )}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged